### ISSUE TITLE
Peer pruning - accept ESTABLISHED,RELATED connections?

### ISSUE THREAD

**Issue by brennen** [tam7t/droplan#11]
Re: [the TODO on pruning peers without causing downtime](https://github.com/tam7t/droplan/blob/master/tables.go#L38), what about the functional equivalent of:

```
iptables -A INPUT -m conntrack --ctstate ESTABLISHED,RELATED -j ACCEPT
```

I don't know that this entirely solves the problem, but it feels like it might mitigate somewhat.  I'm also working with a pretty cursory actual knowledge of iptables here, so feel free to dismiss this notion out of hand. :)

**Comment by tam7t** [comment]
If i understand, you're proposing this rule go above the drop rule on `eth1`:

```
-A INPUT -i eth1 -j DROP
```

so that existing connections are not disrupted? That certainly seems easier than computing a `diff` on the `droplan-peers` chain. It would be nice to put together a little test to verify that this works.

**Comment by brennen** [comment]
Yeah, that was my thought.  I'm not sure it covers all the cases where the firewall might otherwise cause disruption (what happens in the case where there's no existing connection but one is attempted during the window where all connections are blocked?), but it probably somewhat mitigates.

**Comment by tam7t** [comment]
Fixed with #13
